crane/app_util.py

```diff
@@ -71,9 +71,16 @@ def repo_is_authorized(repo_id):
 
     if repo_tuple.protected:
         cert = _get_certificate()
-        if not cert or not cert.check_path(repo_tuple.url_path):
+        logger.info('cert when checking repo %s' % cert)
```

Comment:
Should this (and maybe others) be `logger.debug`?

Reply:
Ideally, this should be logger.debug but since we don't have production environment replica to test this out I have kept it to logger.info so that I could see what has been failed and could correct that.

If we want to keep this as logger.debug, we have to enable debug and restart the crane service in the production environment to track if in case this PR fails to work and if everything goes fine then we have to raise another PR to make log levels back to info and restart the crane service.

I am ok with both the approaches